src/chainparamsbase.cpp

```diff
@@ -17,6 +17,7 @@ const std::string CBaseChainParams::REGTEST = "regtest";
 
 void SetupChainParamsBaseOptions()
 {
+    gArgs.AddArg("-chain=<chain>", "Use the chain <chain> (default: main). Allowed values: main, testnet, regtest", ArgsManager::ALLOW_ANY, OptionsCategory::CHAINPARAMS);
```

Comment:
I don't think `testnet` is an allowed value

```suggestion
    gArgs.AddArg("-chain=<chain>", "Use the chain <chain> (default: main). Allowed values: main, test, regtest", ArgsManager::ALLOW_ANY, OptionsCategory::CHAINPARAMS);
```

Alternatively, you can use 
```cpp
Join({
  CBaseChainParams::MAIN,
  CBaseChainParams::TESTNET,
  CBaseChainParams::REGTEST,
}, ", ");